### PR TITLE
[Twenty Twenty Blocks] Try adding early Block Patterns support to the Twenty Twenty Blocks theme

### PR DESCRIPTION
**Pull request by melchoyce**
This includes one new pattern, based on the Exhibitions grid on https://2020.wordpress.net/.

image

### CODE REVIEW

File: twentytwenty-blocks/functions.php [hunk 1]
```diff
@@ -307,3 +307,16 @@ function twentytwentyblocks_register_styles() {
 	wp_enqueue_style( 'twentytwentyblocks-style', get_stylesheet_uri(), array(), $theme_version );
 }
 add_action( 'wp_enqueue_scripts', 'twentytwentyblocks_register_styles' );
+
+
+/**
+ * Register Block Patterns.
+ */
+
+register_pattern (
```
Comment (jffng) [inline twentytwenty-blocks/functions.php:316]: Let's wrap this function call to be safe / avoid errors if the user is running an older version of Gutenberg.

```php
if ( function_exists( 'register_pattern' ) ) {
  register_pattern (
    ...
  )
}
```

**Comment by kjellr** [comment]
Looks great! Once this incorporates @jffng's suggestion above, I think it's ready to go. 👍

File: twentytwenty-blocks/functions.php (hunk 1 above)
Comment (melchoyce) [reply twentytwenty-blocks/functions.php]: Pushed an update!

File: twentytwenty-blocks/functions.php [hunk 2]
```diff
@@ -313,10 +313,12 @@ function twentytwentyblocks_register_styles() {
...
-    )
-);
\ No newline at end of file
+if ( function_exists( 'register_pattern' ) ) {
+	register_pattern (
+	    'twentytwenty/exhibitions-pattern',
```
Comment (kjellr) [inline twentytwenty-blocks/functions.php:318]:
```suggestion
	    'twentytwenty-blocks/exhibitions-pattern',
```

Super-minor note here, but I think this should match the theme slug.

**Pull request merged**